apps/docs/docs/reference/ui/atoms/Radio.mdx

```diff
@@ -16,10 +18,16 @@ Radios allow users to select one option from a set.
 import { Radio } from '@faststore/ui'
 ```
 
+### Import Styles
+
+```tsx
+import '@faststore/ui/src/components/atoms/Radio/styles.scss'
+```
+
 ## Usage
 
 ```tsx live
-<Radio/>
+<Radio />
 ```
 
 ## Props
```

Comment:
Still need to work on that! 
Props are not sync with `faststore/components`

<img width="862" alt="image" src="https://user-images.githubusercontent.com/3356699/199972320-dd4cdb59-2f53-49bb-8b96-322f3d45e3c6.png">